plutus-core/plutus-core/src/PlutusCore/Core/Instance/Pretty/Readable.hs

```diff
@@ -58,7 +58,7 @@ instance
         TyForall _ name kind body ->
             typeBinderDocM $ \prettyBinding prettyBody ->
                 "all" <+> prettyBinding name kind <> "." <+> prettyBody body
-        TyBuiltin _ builtin       -> unitDocM $ pretty builtin
+        TyBuiltin _ builtin       -> prettyM builtin
```

Comment:
The gist of the change. Previously we'd just call `pretty` purely and now we call the monadic `prettyM` having access to the current context.